profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -112,27 +96,48 @@ function tmgmt_poetry_form_tmgmt_ui_workbench_moderation_translation_review_form
  */
 function tmgmt_poetry_form_tmgmt_job_form_alter(&$form, &$form_state, $form_id) {
   $translator = FALSE;
+
+  // Reference to the translation job object.
+  $job = &$form_state['tmgmt_job'];
+  // Overwriting details which are coming from tmgmt_ui.pages.inc.
+  // Details about job items for give job and sub jobs related to that job.
+  unset($form['job_items_wrapper']['items']);
+  // Inject details only when job was submitted.
+  if (($view = views_get_view('tmgmt_poetry_translation_job_items')) && !$job->isSubmittable()) {
+    $form['job_items_wrapper']['items'] = array(
+      '#type' => 'markup',
+      '#markup' => $view->preview('block', array($job->tjid)),
+      '#prefix' => '<div class="' . 'tmgmt-ui-job-items ' . ('tmgmt-ui-job-manage') . '">',
+      '#attributes' => array(
+        'class' => array(
+          'tmgmt-ui-job-items', 'tmgmt-ui-job-manage',
+        ),
+      ),
+      '#suffix' => '</div>',
+    );
```

Comment:
I'm not sure about `#attributes` on a `markup` element: how they will be treated?

Also, isn't the wrapper taken care by `#prefix` and `#suffix` already?